File: 0657-robot-return-to-origin/solution.c
bool judgeCircle(char* moves) {
    int u=0,r=0;
    for( int i=0; moves[i]!='\0'; i++ )
    {
        if( moves[i] == 'U')
        u++;
        else if(moves[i] == 'D')
        u--;
        else if(moves[i] == 'R')
        r++;
        else if(moves[i] == 'L')
        r--;
    }
    return (u==0 && r==0);
}

File: 0657-robot-return-to-origin/solution.py
class Solution:
    def judgeCircle(self, moves: str) -> bool:
      right=0
      up=0
      for i in moves:
        if i == 'U':
            up += 1
        elif i == 'D':
            up -= 1
        elif i == 'L':
            right -= 1
        else:
            right += 1
      if right == 0 and up == 0:
        return True
      return False
